MobileCenterCrashes/MobileCenterCrashes/MSCrashes.mm

```diff
@@ -188,6 +190,9 @@ + (void)notifyWithUserConfirmation:(MSUserConfirmation)userConfirmation {
     // Get error attachments.
     if ([crashes delegateImplementsAttachmentCallback]) {
       attachments = [crashes.delegate attachmentsWithCrashes:crashes forErrorReport:report];
+      if (attachments.count > kMaxAttachmentsPerCrashReport) {
```

Comment:
This logic should be done within below for loop. We are currently missing validation of attachments, we should call isValid for each attachment before sending. You did this for Android so you should understand. 😄